ui/ts/models/log.ts

```diff
@@ -77,13 +76,22 @@ module Models {
         return url;
       }
 
-      constructor() {
+      constructor(nodeId?: string) {
         this.level(Utils.Format.Severity(2));
         this.max(null);
         this.startTime(null);
         this.endTime(null);
         this.pattern(null);
-        this.node(null);
+        this.node(nodeId);
+        // Add the QueryCache after initialization in order to not fetch without
```

Comment:
can you explain this comment and the code movement it refers to? how could the fetch ever happen during the object's construction?